docs/ota-client-guide/modules/ROOT/pages/garage-sign.adoc

```diff
@@ -0,0 +1,218 @@
+= garage-sign
+
+ifdef::env-github[]
+
+[NOTE]
+====
+We recommend that you link:https://docs.ota.here.com/ota-client/latest/{docname}.html[view this article in our documentation portal]. Not all of our articles render correctly in GitHub.
+====
+endif::[]
+
+
+
+xref:https://github.com/advancedtelematic/ota-tuf/tree/master/cli[garage-sign] is a tool for managing your OTA Connect software repository and securely signing software versions.
+
+In OTA Connect, `garage -sign` is used to:
+
+* Generate new Uptane repo(s) and keys
+* Sign metadata in Uptane repo(s)
+* Add and remove software from repo(s)
+* Pull current metadata from OTA Connect
+* Push local, signed metadata to OTA Connect
+
+
+== Generate new Uptane repository and keys
+
+* *Generate new Uptane repo*
++
+`garage-sign` is used to generate a new repository with the command `garage-sign init`. This initiallizes a new repository, and has the format:

```

Comment:
What does it mean to "initialize a new repository"? Why would I ever want to initialize a new repository when OTA Connect creates one for me?